app.js

```diff
@@ -0,0 +1,56 @@
+// declare variables we need
+const scoreInput = document.querySelector("#score");
+const diceBox = document.querySelector("#dice-box");
+const rollBtn = document.querySelector("#roll-button");
+const playerStatus = document.querySelector("#player-status");
+const diceImg = document.querySelector("#dice-img");
+let score = 0;
+
+diceBox.style.display = "none";
+
+// event listener triggered on button click with condition
+rollBtn.addEventListener("click", () => {
+   if (rollBtn.textContent === "Roll") {
+      let randomNumber = Math.floor(Math.random() * 6 + 1);
+      checkNumber(randomNumber);
```

Comment:
If randomNumber is never used again you could do this in one line `checkNumber(Math.floor(Math.random() * 6 + 1))`

Just remember however maths order of operators (BODMAS), it's often better to be explicity and put parenthesis in place, even if not needed, just for clarity.